resources/js/index.js

```diff
@@ -0,0 +1,57 @@
+import Sortable from 'sortablejs';
+import $ from 'jquery';
+
+document.addEventListener('alpine:initializing', () => {
+    window.Alpine.data('sortableTree', (maxDepth) => ({
+        maxDepth,
+        init() {
+            let nestedSortables = document.getElementsByClassName('js-sortable-group');
+            for (let i = 0; i < nestedSortables.length; i++) {
+                new Sortable(nestedSortables[i], {
+                    group: 'nested',
+                    animation: 150,
+                    fallbackOnBody: true,
+                    swapThreshold: 0.65,
+                    draggable: '.js-sortable-item',
+                    handle: '.js-sortable-handle',
+                    onMove: (evt) => {
+                        if (this.maxDepth >= 0 && this.getDepth(evt.related) > this.maxDepth) {
+                            return false;
+                        }
+                    },
+                    onSort: () => {
+                        this.$wire.sortRows(elementsToArray($('#js-sortable-root-nodes')));
```

Comment:
If this is the only place jQuery is used, you might consider replacing this with native js.